tasks/metrics/mutant.rake

```diff
@@ -19,6 +19,13 @@ namespace :metrics do
       %W[--ignore #{matcher}]
     end
 
+    jobs =
+      if ENV.fetch('DEVTOOLS_SELF', false).eql?('1') && ENV.key?('CIRCLECI')
```

Comment:
I think the whole `DEVTOOLS_SELF` can be removed. Because it only was used to special case mutant zombification in a very historic version. The only thing to branch on is the `CIRCLECI` key in the ENV.